tests/page_test.go

```diff
@@ -1302,3 +1302,70 @@ func TestPageIsVisible(t *testing.T) {
 		})
 	}
 }
+
+func TestPageIsHidden(t *testing.T) {
```

Comment:
This test is very similar to `TestPageIsVisible`. What about merging both? They both are about visibility.

Reply:
Yeah, true, they are very similar, but they're testing two different APIs, so i think it's best to keep them separate.